core/templates/dev/head/pages/exploration_player/correctness_footer_directive.html

```diff
@@ -28,6 +28,40 @@
     padding-left:8px;
   }
 
+  @-webkit-keyframes anim_slides {
+    0% {
+      opacity:0;
+    }
+
+    25% {
+      opacity:0;
+    }
+
+    50% {
+      opacity:0.8;
+    }
+
+    75% {
+      opacity:1;
+    }
+
+    100% {
+      opacity:1;
+    }
+  }
+
+  .inner-container {
+    opacity:0;
+    -webkit-animation-name: anim_slides;
```

Comment:
Perhaps name it `correct_feedback_fade_animation`.

Reply:
Done